### ISSUE TITLE
Cover Binary NIST file with latest standard

### ISSUE THREAD

**Issue by tsaracouvert** [INTERPOL-Innovation-Centre/ANSI-NIST-XML-ITL-Implementation#37]
In order for all our member countries to only refer to our latest standard version even if not using XML format, it is proposed to mention in the standard that the generation of traditional "binary" NIST file can be done based on this standard.
This moreover since the MRT still is covering Traditional fields.

**Comment by tsaracouvert** [comment]
linked to https://github.com/INTERPOL-Innovation-Centre/ANSI-NIST-XML-ITL-Implementation/issues/13

**Comment by hartungdaniel** [comment]
Indeed, it would be good to keep the NIST ITL-1 mother standard and the Interpol Implementation consistent when it comes to serialization methods. 

For example, the frontpage suggests, that the standard is only applicable to XML. Some clarification would be very helpful and appreciated.

Traditional binary encoding is still in use and very important to certain stakeholders, and will likely keep being supported by upcoming NIST ITL-1 standards.

**Comment by tsaracouvert** [comment]
Update the Binary version with the missing part from XML.
Workshop with BKA to organize.

Get an INTERPOL management decision on : 
-end of Binary
-dates

NIST workshop in april 2025 : feedback 
NIST working group on encoding next

**Comment by Jeremy-M-Int** [comment]
Decided:
- The v5.03 document will not be updated
> INTERPOL does not have the means and resources to do so)
- Strong communication on depreciation of v5.03 / Archive it
- We are not including Binary as part of the main document for v6.00 not to promote as an acceptable syntax going forward.
- Strong communication on v6.00 and its positive aspects
- Make an annex to explain Binary on v6.00

Actions:
- [ ] INTERPOL to compare v5.03 and v6.00 Binary versions (e.g. TCN length)
- [ ] INTERPOL to validate and communicate a clear roadmap on the release of BIO HUB handling XML

2 options to export from BIO HUB:
1. Export NIST files to v6.00 Binary NIST Files (referring to Annex & only for retro-compatibility)
3. Export NIST files to v6.00 XML NIST Files
